### PR TITLE
Add hosted reports as vignettes

### LINKED ISSUE #1276: QC: Add hosted versions of standard reports to pkgdown
## QC Details

Let's save a copy of each of our standard reports to the repo and then add links to each in the readme. 

cc: @mattroumaya @samussiah

### PR DESCRIPTION
**Pull request by mattroumaya**
Fix #1276 
- Add standard reports to vignettes
- TODO: Reports don't use CTMS data because I was seeing a bug

To test:
- checkout this branch
- run `pkgdown::build_site()`

### CODE REVIEW

File: README.md
Comment (andersonjake1988) [inline README.md:55]: these links (site and country) are giving 404 errors, when I try and use them, it will work with https://gilead-biostats.github.io/gsm but not when you add /reports or /StandardReportSite.html, not sure if this is just a place holder for now?

**Comment by andersonjake1988** [comment]
Overall looks good, not sure if you wanted to add the QTL stuff into this or leave it just site and country for now until the QTL report is merged and update it later

File: README.md
Comment (mattroumaya) [reply README.md:55]: Good catch! I should have noted this in the PR - this will be resolved once the `pkgdown` site re-deploys after the release.

**Comment by mattroumaya** [comment]
Thanks for the review @andersonjake1988! Let's hold off on adding the QTL reports until we build them out a little bit more. We'll merge in that PR after this PR (+ the release), and can start developing more features for QTL-specific reporting. 

When you get the chance, can you please approve this one? Thanks!

**Review by andersonjake1988 — APPROVED** [review]

**Pull request merged**
